### PR TITLE
Feat templates

### PR DESCRIPTION
**Pull request by anngvu**
Thematically focused on templates for genomics data, especially processed data:

- https://github.com/nf-osi/nf-metadata-dictionary/issues/158
- https://github.com/nf-osi/nf-metadata-dictionary/issues/126
- https://github.com/nf-osi/nf-metadata-dictionary/issues/162
- https://github.com/nf-osi/nf-metadata-dictionary/issues/122 (& other processing-related properties)
- Template documentation updates

### CODE REVIEW

**Comment by anngvu** [comment]
@allaway This is a significant PR, so even though it's open for review, I feel there might still be some changes needed. If you open up the documentation index.html file and go to the Templates section, you'll get a good sense of what the genomics templates updates look like. These updates are a combination of looking at HTAN and e.g. https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-entity-list&anchor=generated_data_file. Let me know what you think so far.

File: docs/index.Rmd [hunk 1]
```diff
@@ -278,6 +310,19 @@ This partial graph view logically relates **data types** to **assays**.
 
 *Documentation currently in development.*
 
+#### Data Level {#Data .tabset .tabset-fade .tabset-pills}
+
+##### Terms
+
+```{r data_subtypes_table, echo=FALSE}
+
+data_level_table <- schema %>%
```
Comment (allaway) [inline docs/index.Rmd:319]: Data Subtype is one of those annotations that I really do not like (in it's current implementation). It's not very explicit. What do you think of adopting more standardized data levels?

Comment (anngvu) [reply docs/index.Rmd:319]: I don't think dataSubtype is all that useful either. You could probably get rid of it and just use dataType with values that capture the needed information. For example, GDC has data type SimpleSomaticMutation vs. AnnotatedSomaticMutation. You have the information that these are processed data files, but moreover Annotated* is an additional level of "processed", which dataSubtype doesn't even help distinguish. But I think that's a little outside of the scope for now so will figure out how to revamp later.

**Comment by allaway** [comment]
## NADIA Curation Report

**Synapse Project:** [syn74317555](https://www.synapse.org/#!Synapse:syn74317555)

### Annotations Applied
| Field | Value |
|-------|-------|
| studyName | EZH1-2 function mostly within canonical PRC2 and exhibit proliferation-dependent redundancy that shapes oncohistone-mediated phenotypes in cancer |
| diseaseFocus | Neurofibromatosis type 1 |
| manifestation | Malignant Peripheral Nerve Sheath Tumor |
| dataType | proteomics |
| assay | label free mass spectrometry |
| species | Homo sapiens |
| tumorType | Malignant Peripheral Nerve Sheath Tumor |
| platform | Other |
| studyLeads | Michel Wassef, Raphaël Margueron |
| institutions | Institut Curie Paris |
| pmid | 30867289 |
| doi | 10.1073/pnas.1814634116 |
| alternateDataRepository | pride.project:PXD012547 |

### Curation Notes
- PRIDE proteomics for PRC2/EZH1-2 cancer cell study; PNAS 2019
- 20 raw mass spec files in Raw Data folder
- studyLeads corrected to first+last author (not data submitter)

**Status:** pendingReview ✓

**Pull request merged**